cmd/commands/git-source.go

```diff
@@ -273,7 +273,15 @@ func createCronExamplePipeline(opts *gitSourceCronExampleOptions) error {
 	sensorFilePath := opts.gsFs.Join(opts.gsCloneOpts.Path(), store.Get().CronExampleSensorFileName)
 
 	eventSource := createCronExampleEventSource()
-	err = opts.gsCloneOpts.FS.WriteYamls(eventSourceFilePath, eventSource)
+	eventSourceRedundanded, err := cleanUpRedundantFieldsFromCalendarEventSource(&eventSource)
```

Comment:
I cannot understand why we are doing this after the `eventSource` was declared, and then we need to `marshal` and `unmarshal` multiple times. Why not just mutate it as an object inside the `createCronExampleEventSource` function?